benchmark/src/bench.rs

```diff
@@ -17,19 +17,19 @@ use std::io::Read;
 use std::sync::atomic::{AtomicBool, Ordering};
 use std::sync::Arc;
 
-use crate::grpc::{
+use futures::{Future, Sink, Stream};
+use grpc::{
     self, ClientStreamingSink, DuplexSink, MessageReader, Method, MethodType, RequestStream,
     RpcContext, RpcStatus, RpcStatusCode, ServerStreamingSink, ServiceBuilder, UnarySink,
     WriteFlags,
 };
-use crate::grpc_proto::testing::messages::{SimpleRequest, SimpleResponse};
-use crate::grpc_proto::testing::services_grpc::BenchmarkService;
-use crate::grpc_proto::util;
-use futures::{Future, Sink, Stream};
+use grpc_proto::testing::BenchmarkService;
+use grpc_proto::testing::{SimpleRequest, SimpleResponse};
+use grpc_proto::util;
 
 fn gen_resp(req: &SimpleRequest) -> SimpleResponse {
     let payload = util::new_payload(req.get_response_size() as usize);
-    let mut resp = SimpleResponse::new();
+    let mut resp = SimpleResponse::new_();
```

Comment:
em, seem `new_` looks strange here, why adding the suffix `_` here 🤔

Reply:
Unfortunately Prost creates a different `new` function for every message, so we need a new name. I chose the closest to the existing one (and one which is easy to search and replace later). Long-term we should probably use `Default::default` instead.